### PR TITLE
Add more sports icons: Padel, Futsal, Handball, Hockey, Badminton, Pelota

### PR DESCRIPTION
**Pull request by dvdmrtnz**
Inspired by https://github.com/organicmaps/organicmaps/pull/2960 I added some more sport icons in this PR.

Related to https://github.com/organicmaps/organicmaps/issues/2956.

Here are the icons:

Padel: (source: using [tennis icon](https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/tennis-m.svg) as a base with design inspired by [Flaticon]( https://www.flaticon.es/icono-premium/padel_1984765))

 

Handball: (source: [Material Icons](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Asports_handball%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048))

 

Futsal: (source: same icon as [soccer icon](https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/soccer-m.svg))

 

Ice Hockey/Field Hockey: (source: [Material Icons](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Asports_hockey%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048))

 

Badminton: (source: using [tennis icon](https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/tennis-m.svg) as a base with design inspired by [Flaticon](https://www.flaticon.es/icono-gratis/badminton_625257))

 

Pelota: (source: using [tennis icon](https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/tennis-m.svg) as a base with design inspired by [Flaticon](https://www.flaticon.es/icono-gratis/pelota-vasca_6938507))

### CODE REVIEW

**Comment by biodranik** [comment]
1. How popular are these sports in OSM?
2. Editor also should support them.

**Comment by dvdmrtnz** [comment]
> 1. How popular are these sports in OSM?

They are quite popular. They all are in the top 50 of [sport=*](https://taginfo.openstreetmap.org/keys/sport#values) tags.

| Tag | Usage | Top | 
|-|-|-|
| [sport=futsal](https://taginfo.openstreetmap.org/tags/sport=futsal) | 7 686 | 24 |
| [sport=badminton](https://taginfo.openstreetmap.org/tags/sport=badminton) | 5 176 | 31 |
| [sport=padel](https://taginfo.openstreetmap.org/tags/sport=padel) | 5 140 | 32 |
| [sport=handball](https://taginfo.openstreetmap.org/tags/sport=handball) | 5 044 | 33 |
| [sport=field_hockey](https://taginfo.openstreetmap.org/tags/sport=field_hockey) | 4 301 | 38 |
| [sport=ice_hockey](https://taginfo.openstreetmap.org/tags/sport=ice_hockey) | 4 170 | 39 |
| [sport=pelota](https://taginfo.openstreetmap.org/tags/sport=pelota) | 3 212 | 46 |

> 2. Editor also should support them.

Are the other sports in the editor? I couln't find them 🤔

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -20986,6 +20986,78 @@
     zh-Hans = 保龄球
     zh-Hant = 保齡球
 
+  [type.sport.padel]
+    en = Padel
+    be = падель
```
Comment (biodranik) [inline data/strings/types_strings.txt:20991]: Падэль

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -20986,6 +20986,78 @@
...
+    en = Futsal
+    be = Футзал
+    de = Futsal
+    es = Futsal
+    eu = Areto-futbola
+    fr = Futsal
+    it = Futsal
+    pt = Futsal
+    ru = Футзал
+    uk = футзал
```
Comment (biodranik) [inline data/strings/types_strings.txt:21011]: Футзал

**Review by biodranik — CHANGES_REQUESTED** [review]
Thanks!

1. Please try to translate to other languages too using `DELIM=' = ' ./tools/unix/translate_categories.sh 'English Name'` script and manually check with DeepL.
2. Please create the issue to add these types to the Editor.

CC @vng

**Comment by dvdmrtnz** [comment]
Done. I also checked with wikipedia

**Review by vng — CHANGES_REQUESTED** [review]
- Merge first and second main commits into one.
- I'd prefer *not* to add this exotic sports into categories.txt (at least now).
- we decided to keep separate types sport=* and leisure=pitch.

File: data/mapcss-mapping.csv [hunk 3]
```diff
@@ -365,8 +365,8 @@ sport|9pin;364;
 sport|10pin;365;
 piste:type|downhill|novice;[piste:type=downhill][piste:difficulty=novice];;name;int_name;366;
 railway|narrow_gauge|bridge;[railway=narrow_gauge][bridge?];;name;int_name;367;
-deprecated|deprecated;368;x
-deprecated|deprecated;369;x
+sport|padel;368;
+leisure|pitch|padel;[leisure=pitch][sport=padel];x;name;int_name;369;leisure|pitch
```
Comment (vng) [inline data/mapcss-mapping.csv:369]: - keep only sport|padel here and below for all other sport types
- move them together somewhere after our current sports block. Less numbers are reserved for possible more frequent types.

**Review by vng — CHANGES_REQUESTED** [review]

File: data/mapcss-mapping.csv (hunk 3 above)
Comment (dvdmrtnz) [reply data/mapcss-mapping.csv]: Done

File: data/categories.txt [hunk 4]
```diff
@@ -7025,6 +7025,222 @@ vi:Nhảy nhót|Bowling
...
+eu:Areto-futbola
+fa:ﻝﺎﺴﺗﻮﻓ
+fi:Futsal
+fr:Futsal
+he:לסטופ
+hu:Futsal
+id:Futsal
+it:Futsal
+ja:フットサル
+ko:"풋살"
```
[the remark this answers is not in the document]
Comment (dvdmrtnz) [reply data/categories.txt]: Done

**Comment by dvdmrtnz** [comment]
> * Merge first and second main commits into one.
Done

> * I'd prefer _not_ to add this exotic sports into categories.txt (at least now).
Removed

> * we decided to keep separate types sport=* and leisure=pitch.
I'm not sure what you mean by this

File: data/mapcss-mapping.csv [hunk 5]
```diff
@@ -458,7 +458,7 @@ highway|living_street|tunnel;[highway=living_street][tunnel?];;name;int_name;457
 deprecated|deprecated;458;x
 deprecated|deprecated;459;x
 deprecated|deprecated;460;x
-sport|handball;461;
+deprecated|deprecated;461;x
```
Comment (vng) [inline data/mapcss-mapping.csv:461]: Wow, we **can't** change numeric values and move existing tags! Return handball back here :)

File: data/mapcss-mapping.csv [hunk 6]
```diff
@@ -1131,20 +1131,20 @@ deprecated|deprecated;1132;x
...
-deprecated|deprecated;1142;x
-deprecated|deprecated;1143;x
-deprecated|deprecated;1144;x
-deprecated|deprecated;1145;x
-deprecated|deprecated;1146;x
-deprecated|deprecated;1147;x
-deprecated|deprecated;1148;x
-deprecated|deprecated;1149;x
+sport|padel;1136;
+leisure|pitch|padel;[leisure=pitch][sport=padel];x;name;int_name;1137;leisure|pitch
```
Comment (vng) [inline data/mapcss-mapping.csv:1135]: leisure|pitch|xxx are not needed here. Keep only sport|xxx types. Remove them.

Existing types like leisure|pitch|tennis are present here for the _backward compatibility_ with old map data.

**Review by vng — CHANGES_REQUESTED** [review]

**Review by vng — APPROVED** [review]

**Pull request merged**